docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,185 @@
+`-` as a binary operator, while `a * -4` treats `*` as a mathematical operation
+and `-` as the negative sign. Hence we can say that the whitespaces plays a
+really important role here, and we use some rules to avoid confusion:
+
+-   There can be no whitespace between a unary operator and its operand.
+-   The whitespace around a binary operator must be consistent: either there is
+    whitespace on both sides or on neither side.
+-   If there is whitespace on neither side of a binary operator, the token
+    before the operator must be an identifier, a literal, or any kind of closing
+    bracket (for example, `)`, `]`, or `}`), and the token after the operator
+    must be an identifier, a literal, or any kind of opening bracket (for
+    example, `(`, `[`, or `{`).
+
+## Details
+
+Symbolic tokens are intended to be used for widely-recognized operators, such as
+the mathematical operators `+`, `*`, `<`, and so on. Those used as operators
+would generally be expected to also be meaningful for some user-defined types,
+and should be candidates for being made overloadable once we support operator
+overloading.
+
+### Symbolic token list
+
+The following is the initial list of symbolic tokens recognized in a Carbon
+source file:
+
+| Token | Explanation                                                                                                |
+| ----- | ---------------------------------------------------------------------------------------------------------- |
+| `*`   | Indirection, multiplication, and forming pointers                                                          |
+| `&`   | Address-of or Bitwise AND                                                                                  |
+| `=`   | Assignment                                                                                                 |
+| `->`  | Return type and `p->x` equivalent to `(*p).x` (in C++)                                                     |
+| `=>`  | Match syntax                                                                                               |
+| `[]`  | Subscript                                                                                                  |
+| `()`  | Function call and function declaration                                                                     |
+| `{}`  | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `,`   | Separate arguments in a function call, elements of a tuple, or parameters of a function declaration        |
+| `.`   | Member access                                                                                              |
+| `:`   | Scope                                                                                                      |
+
+This list is expected to grow over time as more symbolic tokens are required by
+language proposals.
+
+Note: The above list only covers up to
```

Comment:
This should probably be a TODO rather than a note. Or, if you want to fix it right away, the missing tokens I noticed are:
- [Arithmetic operators](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/arithmetic.md)
- [Bitwise operators](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/bitwise.md)
- [Comparison operators](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/comparison_operators.md)
- [`:!`](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/README.md#checked-and-template-parameters)

Reply:
I will be adding all the tokens present here https://github.com/carbon-language/carbon-lang/blob/trunk/explorer/syntax/lexer.lpp

It will cover all these right?